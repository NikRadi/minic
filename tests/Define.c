#define MY_VAR1 10
#define MY_VAR2 20
#define MY_VAR3 (1 + 2)
#define PLUS +

int main() {
    printf("%d %d %d\n", MY_VAR1, MY_VAR2, MY_VAR1 + MY_VAR2);
    printf("%d %d\n", MY_VAR3, MY_VAR3 PLUS MY_VAR3);
}
