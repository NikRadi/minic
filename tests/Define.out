10 20 30
3 6